frontend/src/types/index.ts

```diff
@@ -9,3 +9,13 @@ export type Punchline = {
   new?: boolean;
   viewed?: boolean;
 };
+
+export type Setup = {
+  setup: string;
+  type: "PICK_ONE" | "PICK_TWO" | "DRAW_TWO_PICK_THREE";
```

Comment:
RIP still need to get these other modes working

Reply:
Yep 🥲